test/functional/JLM_Tests/src/org/openj9/test/management/TestOperatingSystemMXBean.java

```diff
@@ -260,8 +303,12 @@ private static boolean test_PhysicalMemoryAPIs(com.ibm.lang.management.Operating
 			 */
 			Assert.assertTrue(totalMemory > 0,
 					"Invalid Physical Memory Info retrieved , Total Physical Memory cannot be 0 bytes.");
-			Assert.assertFalse(freeMemory > totalMemory,
+			if (isPhysicalMemoryTestExcluded()) {
+				logger.info("Testing freeMemory <= totalMemory excluded on this platform");
+			} else {
+				Assert.assertFalse(freeMemory > totalMemory,
```

Comment:
Can you improve the failure message so it includes `freeMemory` and `totalMemory`?